GitHubPolicy.md

```diff
@@ -0,0 +1,177 @@
+
+* Version Control 
+    > This is the process of tracking and managing edits to a codebase. 
+
+* Pull Request
+
+## Best Practices
+[I am a contibutor](#for-contributors)
+
+[I am reviewing a PR](#reviwing-pr)
+
+### **For contributors:**
+* Follow the template.
+    > When creating a pr, the template for the description will autmatically appear (I will add this once we come up with a template). Please follow the instructions on the template.
+
+* Rebase when possible, only merge if absolutely necessary.
+    > In order to keep the history of edits in a linear order, try to rebase from the base branch. Rebasing is typically more complicated, and may sometimes be too difficult. If that is the case, then merge.
+
+* Include translated strings with approriate descriptions.
+
+    > Check that you have included the localization/translation for all strings. Make sure the string can be understood by an external translator who may not have any context. If it is ambiguous, add a description. You can learn more about translations [here]()
+
+* Point the base branch to `develop`, unless you are working on smaller pieces of a bigger project (eg. sprints, new feature).
+    > All pr's should have `develop` as the base branch. The exception is when you are creating smaller pieces of a larger feature. If that is the case, the base branch of this larger feature branch should be `develop`. You can learn more about how to change the base branch [here]()
+
+* Take out all unnecessary comments.
+    > Any comments in the code that do not communicate a meaningful message to another developer should be changed to a meaningful message or taken out of the code. GIVE EXAMPLE
+
+* Code has been tested.
+    > Make sure that you have fully tested your code on a device or emulator. It is up to you as the author, to make sure everything is functioning the way that you intended. 
+
+* Be Timely.
+    > After someone has reviewed you PR and left comments, try to address those comments within 48 hours. This keeps it fresh for yourself and the reviewer, making it easier to understand the comments.
+    <!-- I feel like i need to explain this a little better -->
+
+* Make sure test passes CI.
+    >Our CI tests take a few minutes to run after a PR has been created in Github. Check to make sure that these tests have passed. If they did not pass, see troubleshooting (TODO).
+
+### **Reviwing PR**
+* Don't Fix the code, just comment on Github
+    > If you see any mistakes, just add a comment in github instead of going in and trying to fix it yourself. This keeps the edit history clean and unambigous as to where the contribution came from.
+
+* Be timely
+    > When reviewing, make sure to review the entire PR. Commenting on part of the code, while not looking at other parts of the code may lead to confusion down line. Try to address any replies from the author within 48 hours. This keeps it fresh for yourself and the contributor, making it easier to understand the comments.
```

Comment:
I like the hard number here but wondering if it's we should be that rigid about it vs saying something like "in a timely manner" 🤷‍♂️ guess it depends on how specific we want these guidelines to be

Reply:
Yes! Good point, I am trying to imply that they should TRY. I can change the wording to make it seem more like a suggestion then a hard fast rule